content/articles/transacting-erc20-tokens-brownie/index.md

```diff
@@ -0,0 +1,351 @@
+### Introduction
+ERC20 tokens are a representation of something on the Ethereum Blockchain. They can reference a representation of Gold, in which case those tokens are very valuable. They can also reference a representation of someone's toenails, in which case those tokens are somewhat less valuable.
+
+In this tutorial, you are going to do the following:
+- Create your own ERC20 token
+- Deploy your token to a local blockchain.
+- Make transactions using your token
+- Create a smart contract that interacts with your token.
+
+### Prerequisites
+- Visual Studio Code (VS Code)
+- Brownie python environment installed.
+- Basic knowledge in Python Programming Language.
+- Basic knowledge in Solidity Programming Language.
+
+
+### Goal
+In this article, you will obtain all the knowledge you need to deploy and transact your own ERC20 token.
+
+### Table of contents
+1. Initialization.
+2. Deploy the Token.
+3. Transacting with the token.
+4. Creating a Smart Contract that transacts the Token
```

Comment:
Fix your table of contents so the reader can click and move directly to that section immediately, and add the missing headers. [Example](https://stackoverflow.com/questions/11948245/markdown-to-create-pages-and-table-of-contents).